### ISSUE TITLE
Change DB directory

### ISSUE THREAD

**Issue by JurajNL** [SemkoDev/bolero.fun#67]
Hi,

This is more of a feature request. I have a VPS with Windows on it. I work with two partitions. "C" & "D". So Bolero is saving the DB on C. But now IRI won't sync after a couple of weeks because i'm low on storage on "C". Is there a way to set the DB on an other partition?

Thanks, Juraj

**Comment by stereu** [comment]
You can create a symlink via Commandline. That should work pretty well as workaround.

**Comment by tiker** [comment]
That's exactly what I did.

Create a new directory on D:
Move the entire .bolero directory to the new directory on D:
Then used:
mklink /D oldpath newpath

**Comment by JurajNL** [comment]
Yeah I've done that yesterday. Only it started to sync from start. So db directory was +-20GB after moving. And after the first startup it started with a empty DB. So now we wait and sync :-)

Maybe it'll be a good idea to implement this(directory database location) as an option with the installation. It'll add more to the usability for CarrIOTA Bolero. 

Thanks for the quick responses everyone!
